package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "rzk-1-experimental-highlighting",
-  "displayName": "rzk syntax highlighting",
-  "description": "Syntax Highlighting for rzk, an experimental proof assistant for synthetic ∞-categories",
+  "displayName": "Rzk language server",
```

Comment:
For the name I would suggest simply "Rzk". This is the style followed by most language extensions (like [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python), [Haskell](https://marketplace.visualstudio.com/items?itemName=haskell.haskell), [Dart](https://marketplace.visualstudio.com/items?itemName=Dart-Code.dart-code), ...), including other proof assistants (such as [lean4](https://marketplace.visualstudio.com/items?itemName=leanprover.lean4) and [coq](https://marketplace.visualstudio.com/items?itemName=ruoz.coq))